azurerm/provider.go

```diff
@@ -144,6 +144,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_availability_set":                        resourceArmAvailabilitySet(),
 			"azurerm_cdn_endpoint":                            resourceArmCdnEndpoint(),
 			"azurerm_cdn_profile":                             resourceArmCdnProfile(),
+			"azurerm_cognitive_account":                       resourceArmCognitiveAccount(),
```

Comment:
@katbyte I'm not sure if this should be `azurerm_cognitive_services_account` - WDYT?

Reply:
Unless there could be a `azurerm_cognitive_services_account` and a `azurerm_cognitive_account` i think the services part noise and would prefer it as is.